### ISSUE TITLE
[Ch 14][EPUB][Major] λ 가 `$\lambda$` 문자 그대로 렌더링 — 절 제목·표 캡션 포함 6곳

### ISSUE THREAD

**Issue by sy2399** [yoon-gu/neuqes-101#137]
## 대상

- 기준 커밋: `f3453ab` (origin/master)
- 파일: `book/chapters/ch14_auxiliary_loss.tex` (6곳)
- 환경: 정적 검토 + `executed/` 대조 + **EPUB(pandoc 3.10.1) · PDF(XeTeX/TeX Live 2026) 실제 빌드**. Colab 재실행 미수행

## 실제 결과

EPUB·PDF에서 λ가 **`$\lambda$` 문자 그대로** 보입니다.

```
손실 노트 — Combined loss L = L_main + $\lambda$ · L_aux        ← 절 제목
표: 14장 손실 노트 - Combined loss L = L_main + $\lambda$ · L_aux 표  ← 표 캡션
BCE per-label + $\lambda$·MSE                                   ← 변화추적표 Loss 열
L_main + $\lambda$·L_aux 직접 계산                               ← 변경점 표 결합 loss 행
```

tex 소스에는 이렇게 들어 있습니다.

```latex
% 96행
\textbf{\texttt{BCE\ per-label\ +\ \$\textbackslash{}lambda\$·MSE}}
% 248행 — 절 제목
\section{\texorpdfstring{손실 노트 --- Combined loss \texttt{L\ =\ L\_main\ +\ \$\textbackslash{}lambda\$\ ·\ L\_aux}}{...}}
```

`\$` = 리터럴 달러, `\textbackslash{}` = 리터럴 백슬래시라 독자에게 `$\lambda$` 로 노출됩니다.

## 기대 결과

`λ` 로 보여야 합니다. 노트북과 WikiDocs 는 이미 그렇습니다.

## 재현 절차

```bash
# EPUB
cd book && ./build-epub.sh
unzip -q build/epub/*.epub -d /tmp/ex && grep -c '\$\\lambda\$' /tmp/ex/EPUB/text/ch017.xhtml   # → 6

# PDF
cd book && latexmk main.tex
pdftotext build/neuqes-101-ch01-34-manuscript.pdf - | grep -c '\$\\lambda\$'                    # → 24 (전 챕터)
```

## 로그

```
$ grep -n 'textbackslash' book/chapters/ch14_auxiliary_loss.tex
96:  ... \texttt{BCE\ per-label\ +\ \$\textbackslash{}lambda\$·MSE} ...
123: ... \texttt{L\_main\ +\ \$\textbackslash{}lambda\$·L\_aux}} 직접 계산 ...
248: \section{\texorpdfstring{손실 노트 --- Combined loss \texttt{...\$\textbackslash{}lambda\$...}}{...}}
269: \begin{booktable}{14장 손실 노트 - Combined loss L = L\_main + \$\textbackslash lambda\$ · L\_aux 표}{tab:ch14-05}
290: \begin{booktable}{... 같은 캡션 ...}{tab:ch14-06}
```

768·778·886행에도 `\textbackslash lambda` 가 있으나 이쪽은 `\texorpdfstring{\(\lambda\)}{...}` 의 **둘째 인자(PDF 북마크 대체 텍스트)** 라 본문 렌더링에는 영향이 없습니다. 이번 건에서 제외합니다.

## 추가 확인 — 근본 원인

`book/tools/notebook_to_tex.py:2012`

```python
text.replace("λ", r"$\lambda$")
```

pandoc 에 넘기기 전 정규화에서 λ 를 `$\lambda$` 로 바꿉니다. **산문에서는 맞지만 백틱 코드 스팬 안에서는 `$...$` 가 수식이 아니라서** pandoc 이 리터럴로 이스케이프합니다. 노트북 원문은 처음부터 `λ` 문자였습니다 (`d454f1f`).

```
노트북      | **`BCE per-label + λ·MSE`**      ✅
WikiDocs    | `BCE per-label + λ·MSE`          ✅
tex/EPUB    | BCE per-label + $\lambda$·MSE    ❌
```

## 전파 범위

각 산출물을 직접 열어 확인했습니다.

| 축 | 상태 |
|---|---|
| 📓 Notebook | **정상** — `λ` 문자 |
| 📄 WikiDocs `pages/14-*.md` | **정상** — `λ` 및 `$$…\lambda…$$` 수식 블록 |
| 📕 EPUB `ch14_auxiliary_loss.tex` | **결함 6곳** |

**같은 원인으로 다른 챕터에도 있습니다** (이번 이슈 범위 밖 — 각 챕터 담당자용 참고).

| tex | 건수 |
|---|---|
| ch13_bert_multilabel | 2 |
| ch17_ko_multilabel | 2 |
| ch18_ko_auxiliary | 6 |
| ch19 · ch20 · ch21 | 각 1 |

EPUB 전체 19회 / PDF 24회.

## 심각도

**Major** — 챕터의 핵심 공식(`L = L_main + λ·L_aux`)이 절 제목·표 캡션에서 깨져 나옵니다. 이 챕터는 Loss 축의 클라이맥스라 λ 표기가 내용의 중심입니다.

## 담당

**재실행 불필요 · 재변환 불필요 · tex 손수정 후 EPUB/PDF 재빌드 필요.**

⚠️ **`notebook_to_tex.py` 재생성으로 고치지 마세요.** ① master `f3453ab` 의 compact 원고를 옛 장문 포맷으로 되돌립니다 ② 위 λ 치환 로직이 그대로라 같은 손상이 재발합니다. tex 손수정이 맞습니다.

Related to #38

**Comment by sy2399** [comment]
## 수정 완료 — @sy2399 (슬롯 b2), 2026-08-24

브랜치 `fix/ch14-review-2026-09-b2` · 커밋 `f472f9c` + `307e0fc` + `4daeb96`

**세 번에 나눠 고쳤습니다 — 같은 근본 원인인데 이스케이프 형태가 셋이었습니다.**

| 커밋 | 대상 | 형태 |
|---|---|---|
| `f472f9c` | 96·123·248·269·290행 (6곳) | `\$\textbackslash{}lambda\$` / `\$\textbackslash lambda\$` |
| `307e0fc` | 269·290 재수정 | 리터럴 λ 가 본문 폰트에서 두부 — 아래 참조 |
| `4daeb96` | 530행 | `$\lambda$` (verbatim 안이라 이스케이프 안 됨 — grep 에 안 걸렸음) |

### 두 번째 커밋이 필요했던 이유

리터럴 λ 로 바꿨더니 **표 캡션 2곳에서 두부**가 났습니다.

```
Missing character: There is no λ (U+03BB) in font [NanumMyeongjo-Regular]
```

- 모노 **NanumGothicCoding** — λ **있음** ✅ → `\texttt{}`·`lstlisting` 안(96·123·248·530)은 문제없음
- 본문 **NanumMyeongjo** — λ **없음** ❌ → `\begin{booktable}{…}` 캡션(269·290)이 두부

캡션은 `\captionof` 로 이동 인자에 들어가므로 이동 인자에서도 안전한 `\ensuremath{\lambda}` 를 썼습니다.

### 남긴 것

- **768·778·886행** — `\texorpdfstring{\(\lambda\)}{…\textbackslash lambda…}` 의 **둘째 인자(PDF 북마크)**. 첫 인자가 정상 수식이라 본문 렌더링에 영향 없음
- **1095행** `\onlineAppendixLink{…}{$\lambda=0.05$에서…}` — verbatim 밖의 **진짜 수식**이라 정상 렌더링

### 검증

| | 수정 전 | 수정 후 |
|---|---|---|
| EPUB `ch017.xhtml`(Ch 14) | 6회 | **0회** |
| PDF 두부(U+FFFD) | 0 | **0** |
| `Missing character` 경고 | 0 | **0** |

PDF 실제 출력 확인:
```
14.4. 손실 노트 — Combined loss L = L_main + λ ・ L_aux        ← 절 제목
BCE per-label + λ・MSE                                        ← 변화추적표
L_main + λ・L_aux 직접 계산                                    ← 변경점 표
14장 손실 노트 - Combined loss L = L_main + λ · L_aux 표        ← 표 목록
loss = outputs.loss + λ · MSE(                                ← 코드 요약
```

### 범위 밖으로 남긴 것

같은 원인이 ch13(2)·ch17(2)·ch18(6)·ch19·20·21(각1) 에도 있습니다. EPUB 잔여 13회 / PDF 15회 — 각 챕터 담당자 몫입니다.

⚠️ 근본 원인 `notebook_to_tex.py:2012` 의 `text.replace("λ", r"$\lambda$")` 는 그대로입니다. **재생성하면 재발합니다.**

재실행·재변환 불필요 · EPUB/PDF 재빌드 완료.

**Comment by jwrim9305** [comment]
수정 확인했습니다. 확인 브랜치: `fix/ch14-review-2026-09-b2` (`4daeb96`)

**tex 에서 리터럴 기호가 모두 사라졌습니다.**

```
리터럴 $\textbackslash{}...$ 잔여 : 0
정상 λ                        : 6
표 캡션 \ensuremath{\lambda}   : 2
```

네 가지 이스케이프 형태(`\inlinecode`·`\texttt`·`lstlisting`·표 캡션)를 모두 잡으셨고, 캡션만 `\ensuremath` 로 분리한 처리도 폰트 지원(NanumMyeongjo 에 λ 없음)에 맞습니다.

| 산출물 | 반영 | 재실행/재생성 |
|---|---|---|
| 📓 노트북 | 해당 없음 | 원천은 정상 λ · 재실행 불필요 |
| 📄 WikiDocs | 해당 없음 | 페이지도 정상 λ · 재변환 불필요 |
| 📕 EPUB(tex) | ✅ 수정됨 | **재빌드 필요** (재생성 시 되돌아감 — 아래 참고) |

**⚠️ 머지 후 작업 순서 주의.** 이 수정은 tex 손수정이라 **재실행 → tex 재생성 단계에서 전부 되돌아갑니다.** 종결 코멘트에 적어 주신 그대로입니다. 근본 원인을 #169 로 등록했으니, 도구 수정이 먼저 들어간 뒤 tex 를 재생성하면 손수정 없이 해소됩니다. 순서가 반대가 되면 이 6곳을 다시 손으로 고쳐야 합니다.

**Comment by jwrim9305** [comment]
수정 확인했습니다. 확인 브랜치: `fix/ch14-review-2026-09-b2` (`4daeb96`)

**tex 에서 리터럴 기호가 모두 사라졌습니다.**

```
리터럴 $\textbackslash{}...$ 잔여 : 0
정상 λ                        : 6
표 캡션 \ensuremath{\lambda}   : 2
```

네 가지 이스케이프 형태(`\inlinecode`·`\texttt`·`lstlisting`·표 캡션)를 모두 잡으셨고, 캡션만 `\ensuremath` 로 분리한 처리도 폰트 지원(NanumMyeongjo 에 λ 없음)에 맞습니다.

| 산출물 | 반영 | 비고 |
|---|---|---|
| 📓 노트북 | 해당 없음 | 원천은 정상 λ |
| 📄 WikiDocs | 해당 없음 | 페이지도 정상 λ |
| 📕 EPUB(tex) | ✅ 수정됨 | **재빌드 필요** |

**⚠️ 머지 후 작업 순서 주의.** 이 수정은 tex 손수정이라 **재실행 → tex 재생성 단계에서 전부 되돌아갑니다.** 종결 코멘트에 적어 주신 그대로입니다. 근본 원인을 #169 로 등록했으니, 도구 수정이 먼저 들어간 뒤 tex 를 재생성하면 손수정 없이 해소됩니다. 순서가 반대가 되면 이 6곳을 다시 손으로 고쳐야 합니다.
